app/controllers/articles_controller.rb

```diff
@@ -224,10 +227,11 @@ def handle_user_or_organization_feed
   end
 
   def handle_tag_feed
-    @tag = Tag.aliased_name(params[:tag])
-    return unless @tag
+    tag_name = Tag.aliased_name(params[:tag])
+    return unless tag_name
 
-    @articles = @articles.cached_tagged_with(@tag)
+    @tag = Tag.find_by(name: tag_name)
+    @articles = @articles.cached_tagged_with(tag_name)
```

Comment:
Would `@tag.name` also work?  Also, it looks like we're changing the type of `@tag` from a `String` to a `Tag`.  My assumption is we're not using `@tag` elsewhere.

Reply:
I'd like to pass `@tag` to `feed.rss.builder` as a `Tag` rather than just the name so I can reference other parts of the `Tag` like its `url`.  Looks like other places in the module `@tag` is used as a `Tag`, and the feed was the odd one out, where `@tag` was a `String`.

I used `tag_name` rather than `@tag.name` in line 234 because I already have the value cached as a local variable anyways thanks to line 230 so I thought I might as well use it.

I briefly looked into changing the call to `Tag.aliased_name` to something that returns an actual tag, but I was getting tangled up in the `Tag` model and decided to simplify my changes.

Anyways, let me know if you have a problem with any of this.